test/sqlsmith/mzcompose.py

```diff
@@ -99,40 +99,56 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         # Very simple data for our workload
         c.sql(
             """
-            CREATE SOURCE tpch
-              FROM LOAD GENERATOR TPCH (SCALE FACTOR 0.00001);
```

Comment:
TPCH was removed because it involves something that is no longer supported, or just to vary the workload?

Reply:
The load generators haven't found anything of interest in SQLsmith.